db/seeds.rb

```diff
@@ -1,35 +1,47 @@
+  park_activity = ''
+  park_facility = Park.all.last.facility # most recently created Park object's facility attribute
+
+  case park_facility
+  when 'Ice Skating Rinks'
+    park_activity = 'Ice Skating'
+  when 'Bocce Courts'
+    park_activity = 'Bocce'
+  when 'Basketball Courts'
+    park_activity = 'Basketball'
+  when 'Tennis Courts'
+    park_activity = 'Tennis'
+  when 'Football Fields'
+    park_activity = 'Football'
+  when 'Baseball Fields'
+    park_activity = 'Baseball'
+  end
+
+  activity = Activity.find_or_create_by(:name => park_activity)
+  ActivityPark.create(:park_id => park.id, :activity_id => activity.id)
 end
 
-User.create(first_name: "Leslie", last_name: "Knope", email: "leslie@pawneeparks.gov", zipcode: "11217", profile_picture: "http://38.media.tumblr.com/avatar_c964046726bd_128.png", favorite_games: "Basketball, Football, Tennis", host: false, provider: "facebook", uid: nil, token: nil, expires_at: nil, expires: true)
-User.create(first_name: "Ron", last_name: "Swanson", email: "ron@pawneeparks.gov", zipcode: "11211", profile_picture: "http://38.media.tumblr.com/avatar_c61787a2f2a8_128.png", favorite_games: "Ice Skating, Tennis", host: false, provider: "facebook", uid: nil, token: nil, expires_at: nil, expires: true)
-User.create(first_name: "Tom", last_name: "Haverford", email: "tom@pawneeparks.gov", zipcode: "10001", profile_picture: "http://33.media.tumblr.com/avatar_f0eda096578c_128.png", favorite_games: "Bocce, Tennis, Swimming", host: true, provider: "facebook", uid: nil, token: nil, expires_at: nil, expires: true)
-User.create(first_name: "April", last_name: "Ludgate", email: "april@pawneeparks.gov", zipcode: "10001", profile_picture: "https://fbcdn-profile-a.akamaihd.net/hprofile-ak-ash2/v/t1.0-1/c14.14.172.172/s50x50/532819_214610251993778_13027322_n.jpg?oh=0fe009be50955fd755b792966ce2abc0&oe=55B0D91D&__gda__=1437733086_5dba88b8e2d8915639d2961c381c61c4", favorite_games: "Basketball", host: false, provider: "facebook", uid: nil, token: nil, expires_at: nil, expires: true)
-User.create(first_name: "Andy", last_name: "Dwyer", email: "andy@pawneeparks.gov", zipcode: "10001", profile_picture: "https://33.media.tumblr.com/avatar_50a6b2708aaa_128.png", favorite_games: "Basketball", host: false, provider: "facebook", uid: nil, token: nil, expires_at: nil, expires: true) 
-User.create(first_name: "Ben", last_name: "Wyatt", email: "ben@pawneeparks.gov", zipcode: "11215", profile_picture: "http://33.media.tumblr.com/avatar_4b3f765f50c0_64.png", favorite_games: "Baseball, Tennis, Swimming, Football", host: true, provider: "facebook", uid: nil, token: nil, expires_at: nil, expires: true)
+User.create(first_name: 'Leslie', last_name: 'Knope', email: 'leslie@pawneeparks.gov', zipcode: '11217', profile_picture: 'http://38.media.tumblr.com/avatar_c964046726bd_128.png', favorite_games: 'Basketball, Football, Tennis', host: false, provider: 'facebook', uid: nil, token: nil)
```

Comment:
Line is too long. [284/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.